test/unit/spec/PasswordExpired_spec.js

```diff
@@ -124,14 +128,57 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordExpiredForm, Beacon, Expec
             username: 'inca@clouditude.net',
             password: 'newpwd'
           });
+          return tick();
```

Comment:
Calling out that the last comment I added replaced the tick() with a waitForSpy call